README.md

```diff
@@ -142,25 +142,16 @@ py -3 -m venv ./.venv
      ```bash
      python robot.py sim
      ```
-     python -m robotpy_installer install-python
-
-   ```
-
-   ```
-
-1. **Upload robotpy modules to roboRIO**
-   (must be connected to roboRIO)
-   ```bash
-   python -m robotpy_installer install robotpy
-   ```
-   (examples: `robotpy`, `robotpy[ctre,navx]`, `robotpy[all]`) (see: [robotpy on pypi](https://pypi.org/project/robotpy/))
-1. **Deploy robotpy program**
-   - To robot
-     (must be connected to roboRIO)
-     ```bash
-     python robot.py deploy
-     ```
-   - To simulator
-     ```bash
-     python robot.py sim
-     ```
+#### Update PIP requirements
+If there are RobotPy updates we want to upgrade our requirements.txt file to match after verifying it works.
+1. **Install latest version**
+    ```bash
+    pip install --upgrade -r requirements-latest.txt
```

Comment:
Could this be run without the need for a latest file? I don’t understand the purposed of “latest”

Reply:
As far as I can tell there is no "upgrade all" option in pip.  You either need to specify a package to upgrade or a requirements file to upgrade.  Given the main requirements file has pinned versions it can't be used for upgrades.  To be honest I didn't like this, there has to be a better way, but I didn't see any  in my short amount of searching.